### PR TITLE
Added working Nameplates (one while looking on another player)

### PR DESCRIPTION
**Pull request by Alf21**
image

Just a simple nameplate.

### CODE REVIEW

File: code/ui/Nameplate.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+using Sandbox.UI;
+using Sandbox.UI.Construct;
+
+using TTTReborn.Player;
+
+namespace TTTReborn.UI
+{
+	public class Nameplate : Panel
+	{
+		public float MaxDrawDistance = 500;
```
Comment (mzegar) [inline code/ui/Nameplate.cs:11]: this can probably be static private.

File: code/ui/Nameplate.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+        private Label nameLabel;
```
Comment (mzegar) [inline code/ui/Nameplate.cs:13]: This is indented an extra space in.

File: code/ui/Nameplate.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+		public Nameplate()
+		{
+			StyleSheet.Load("/ui/Nameplate.scss");
+
+            nameLabel = Add.Label("", "name");
```
Comment (mzegar) [inline code/ui/Nameplate.cs:19]: This is indented an extra space in aswell.

**Review by mzegar — APPROVED** [review]
Quick comments, personally, I don't like the black border around the name. It takes up a bit too much of the screen.

We can discuss that at a later point though.

**Comment by Westsaid** [comment]
I agree with mat, I think the border is not necessary in this small panel and can be removed

**Review by Westsaid — APPROVED** [review]
Good and simple work!

I got 3 style changes Ill commit afterward, tell me if you are ok with them:
- Remove border 
- Set font-weight : 400
- Half the size of the padding-y

Before: 
image

After:
image

**Pull request merged**